src/remediations/controller.read.js

```diff
@@ -27,6 +27,33 @@ const catchErrorCode = (code, fn) => e => {
     throw e;
 };
 
+async function fetchRemediation (req) {
+    const id = req.params.id;
+    const hosts = req.query.hosts;
+
+    if (_.isUndefined(req.user)) {
+        const batchProfileInfo = await inventory.getSystemProfileBatch(hosts); // TODO: change batchProfileInfo var name
+
+        if (_.isEmpty(batchProfileInfo)) {
+            return [];
```

Comment:
why `[]`?

Reply:
Ah, yes this is redundant, I'll return batchProfileInfo since it's already empty.

Reply:
This was meant to trigger the "404" error in the playbooks function by returning an "empty" remediation but It would probably make more sense to throw that 404 error here.

Reply:
Is it going to 404, though? Walking through the code it seems to me that it's going to choke on `remediation.toJSON()` instead.